types/meteor/globals/mongo.d.ts

```diff
@@ -125,44 +125,46 @@ declare module Mongo {
         [id: string]: Number;
     }
 
+    type TransformFn<T> = ((doc: T) => any) | null | undefined;
+
+    type Options<T> = {
+        sort?: SortSpecifier;
+        skip?: number;
+        limit?: number;
+        fields?: FieldSpecifier;
+        reactive?: boolean;
+        transform?: TransformFn<T>;
+    }
+
+    type DispatchTransform<Transf, DefType, U> = Transf extends (...args: any) => any ? ReturnType<Transf> : Transf extends null ? DefType : U;
```

Comment:
As I wrote in one of my previous reviews, I think this really deserves better naming. How about `Transform`, `T`, `U`? It'll also match the `Collection<T, U>`. Having said that, `Transform` could be used in place of `Fn` in other functions as well.